e2e/cypress/integration/channel_settings/more_unreads_position_with_scroll_spec.js

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @channel_settings
+
+describe('Channel settings', () => {
+    let mainUser;
+    let otherUser;
+    let myTeam;
+
+    // This will ensure a list of channel names that will be alphabetically sorted
+    const channelNames = new Array(20).fill(1).map((value, index) => `scroll${index}`);
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user: firstUser}) => {
+            mainUser = firstUser;
+            myTeam = team;
+
+            cy.apiCreateUser().then(({user: secondUser}) => {
+                otherUser = secondUser;
+                cy.apiAddUserToTeam(team.id, secondUser.id);
+            });
+
+            cy.wrap(channelNames).each((name) => {
```

Comment:
Please add a one line comment to clarify what is happening below (this will help QA analysts and others who are not directly involved in writing Cypress test to understand how the test is structured).

Reply:
Happy to oblige ! :) Will fix